package.json

```diff
@@ -7,7 +7,8 @@
   "main": "./lib/index.js",
   "exports": {
     ".": "./lib/index.js",
-    "./dom/.": "./lib/dom/index.js",
+    "./dom": "./lib/dom/index.js",
+    "./internal/base-component": "./lib/internal/base-component/index.js",
```

Comment:
This seems like a possible anti pattern to have it in an internal folder but then export it publicly. Can we not rather have a more public export in the main entry point?

Reply:
We are not ready yet to call it a stable API (also not beta, because no future plans making it public).

Do you have any other suggestions how share this code across packages?

Reply:
Agreed to proceed with the change.